warehouse/redshift/redshift.go

```diff
@@ -40,6 +40,18 @@ type HandleT struct {
 	Upload        warehouseutils.UploadT
 }
 
+// String constants for redshift destination config
+const (
+	AWSAccessKey        = "accessKey"
+	AWSAccessSecret     = "accessKeyID"
```

Comment:
Any plan to change this accessKeyID to something like accessKeySecret?

If this is a breaking change, we can keep it the way it is now, and write a comment that we won't change.